### ISSUE TITLE
spago doesn't like development versions of purs

### ISSUE THREAD

**Issue by btrepp** [spacchetti/spago#116]
So I grabbed purs 0.12.3 and stack installed it (no windows release it seems).

`spago install` gives
```
spago: Failed to parse 'purs --version' output:
"0.12.3 [development build; commit: b35412a9ea49a46d315a0214cd2c3908442ab7a7]
"
```

Might be nice if there is some sort of flag/escape hatch for "It may not be compatible but I know what I'm doing"

**Comment by f-f** [comment]
@btrepp fix should be up in #117, could you try it?

Re having a "general escape hatch": it's hard to propagate this kind of thing all over the place, as the failure modes are different. E.g. in this case we could log a warning and return a `Maybe` instead of failing, and then propagate the general handling around the rest of the program that needs it (in this case in the package-set version check). Though this fix should be good enough to support all the versions we might get

**Comment by f-f** [comment]
Update: now we don't fail if parsing fails, but just leave a warning
